src/components/Switch/Switch.tsx

```diff
@@ -0,0 +1,60 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { Size } from "@utilities/enum";
+import generateRandomId from "@utilities/generateRandomId";
+import { merge } from "@utilities/merge";
+import { MouseEvent, ReactElement } from "react";
+import Label from "../Form/Label/Label";
+import css from "./Switch.module.css";
+
+export type SwitchProps = {
+    on?: boolean;
+    disabled?: boolean;
+    label?: string;
+    name?: string;
+    size?: Size.Large | Size.Small;
+    onChange?: (e: MouseEvent) => void;
```

Comment:
```suggestion
    onChange?: (event: MouseEvent) => void;
```
nit: we decided to not use the shorthand `e` but `event` everywhere

Reply:
@SamuelAlev are things like that documented somewhere like coding guidelines? cc @dwirz